adoc/Harbor-Private-Registry-Troubleshooting.adoc

```diff
@@ -0,0 +1,59 @@
+[#troubleshooting]
+== Troubleshooting
+
+The installation using Helm is fully automated and should succeed when the requirements are met.
+After a few seconds, the ingress controller configured path should be responding with the login screen of an admin UI, see the <<administration>> section.
+
+In case of issues, please make sure all pods in registry namespace are running and all deployments are in the Ready state.
+If a pod fails to start, investigate the situation with your debugging tool of choice (link:https://k9scli.io/[k9s], or kubectl).
+
+Private registry deploys in the given {kube} namespace multiple deployments and one statefulset, and all of them should be healthy.
+
+=== Possible problems
+
+* As mentioned in the requirements, ingress-controller needs to be set up in your {kube} cluster
+* When accessing the web UI, the browser complains about unknown certificate.
++
+Check the <<install-tls-security>> for a solution.
+* Images for various {spr} components cannot be found, and Pods are not getting created.
++
+Make sure registry.suse.com is accessible from your network.
+* Pods are not starting, most of them fail with failed to connect to database pod error.
++
+Check the errors in the database pod. Does it complain with something like FailedScheduling <unknown> default-scheduler pod has unbound immediate `PersistentVolumeClaims` (repeated 2 times)? Possible reason is that the `storageClass` is missing or no `storageClass` is marked as default.
+
```

Comment:
The error line (_FailedScheduling <unknown> default-scheduler pod has unbound immediate `PersistentVolumeClaims` (repeated 2 times)?_) should be formatted separately from the paragraph text...